src/science/soilsnow/cbl_remove_trans.F90

```diff
@@ -31,14 +31,14 @@ SUBROUTINE remove_trans(dels, soil, ssnow, canopy, veg)
              ! which can be removed:
              xx = canopy%fevc * dels / CHL * veg%froot(:,k) + diff(:,k-1)   ! kg/m2
              diff(:,k) = MAX( 0.0_r_2, ssnow%wb(:,k) - soil%swilt) &      ! m3/m3
-                  * soil%zse(k)*1000.0
+                  * soil%zse(k)*Cdensity_liq
```

Comment:
I'm going to create another issue for this - since the evaluation of `diff` doesn't recognise that only liquid water can be transpired we risk creating a water imbalance by forcing `wb` down so that `wbice/`wb > frozen_limit`.

This issue has always been in the code - so not part of this PR - but it's potentially made worse by allowing ice to have a different density to liquid water.